components/SignIn.tsx

```diff
@@ -0,0 +1,53 @@
+import tw from 'twrnc';
+import { Link } from 'expo-router';
+import { Text, TextInput, View, Pressable } from 'react-native';
+import React from 'react';
+
+function SignIn() {
+  const [email, setEmail] = React.useState('');
+  const [password, setPassword] = React.useState('');
+
+  const inputStyle = tw`bg-white px-3.5 py-3 rounded-2xl`;
+
+  const callApi = () => {
+    console.log('Email: ' + email);
+    console.log('Password: ' + password);
+
+    setEmail('');
+    setPassword('');
+  };
+
+  return (
+    <>
+      <View style={tw`px-4`}>
```

Comment:
Folosim `<ScrollView />`, in caz ca ecranul este prea mic sa se poata da scroll pentru a vedea tot continutul.